site/content/docs/5.0/components/carousel.md

```diff
@@ -206,6 +206,34 @@ Add `data-bs-interval=""` to a `.carousel-item` to change the amount of time to
 </div>
 {{< /example >}}
 
+### Suppressing touch swiping
+
+By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute.
+
+{{< example >}}
+<div id="carouselExampleControlsNoTouching" class="carousel slide" data-bs-ride="carousel" data-touch="false">
```

Comment:
`data-touch` does not contain bs namespace here, and attributes without bs namespace are ignored by Bootstrap.

Reply:
aha! perhaps that's all there is to it? (though that wouldn't explain the report from users saying it's also causing issues in 4.5)

Reply:
NVM, I forgot it is for V4